### ISSUE TITLE
Changing image transform handle directly in `{myImgTransform}.yaml` file causes Asset settings page for given image transform NOT FOUND[4.x]: 

### ISSUE THREAD

**Issue by dominikkrulak** [craftcms/cms#13338]
### Description
When I changed image transform handle in `{myImgTransform}.yaml` file and than applied project config YAML changes I've noticed that the CP's Settings -> Assets -> Image Transforms -> {myImgTransform} setting link, gives 404 status code.

### Steps to reproduce

1. Directly change image transform handle  for any image transform in your  `config/project/imageTransforms/{yourImgTransform}.yaml` file
2. Apply changes to project config
3. Go to CP -> Settings -> Assets -> Image Transforms -> {yourImgTransform} setting link

### Expected behavior
Should load the setting page

### Actual behavior
404, Page not found

### Craft CMS version

4.4.14

### PHP version

8.2.5

### Operating system and version

Linux 5.15.49-linuxkit

### Database type and version

MySQL 8.0.33

### Image driver and version

Imagick 3.7.0 (ImageMagick 6.9.11-60)

### Installed plugins and versions

CKEditor | 3.4.0
Contact Form | 3.0.1
Contact Form Honeypot | 2.0.0
Feed Me | 5.1.4
Mailgun | 3.0.0
4.0.0-beta.2
Minify | 3.7.12
Neo | 3.0.4
SEO | 4.1.2
Store Hours | 3.0.0
Super Table | 3.0.9
Vite | 4.0.6

**Comment by dominikkrulak** [comment]
Hi @i-just, no I'm not able to access the transform via the old transform handle.

- Old transform handle `portrait4to5_768x960`
```
{# Can not access it #}
{{ image.getUrl('portrait4to5_768x960') }} 
```
- New transform handle `4to5_768x960`
```
{# Can access it #}
{{ image.getUrl('4to5_768x960') }} 
```
May it relate to some cache layer perhaps?

**Comment by i-just** [comment]
Thanks for confirming. Did you try clearing the cache (Control Panel > Utilities > Caches)?

Also, I should mention that it’s not advised to change the yaml files directly. More info can be found at the bottom of the usage section: https://craftcms.com/docs/4.x/project-config.html#usage

**Comment by dominikkrulak** [comment]
Yes I did cleared the cache. 

Yes I read that but it seemed easier to edit `yaml` files at once than from control panel because I have quiet "few" image transform handles.

I also use Redis cache and I tried to search for a key that may relate to image transforms that is not purged or overwritten correctly but didn't find anything related. It doesn't bugs me at all because image transforms work as expected but I though I should report that. So I turned up to opening the issue.

I also recorded [a video](https://www.veed.io/view/becd99e0-01cc-4ddb-a4d9-33e511be9f94?panel=share) of the issue reproduction.
